omod/src/main/resources/config.xml

```diff
@@ -382,6 +382,14 @@
 			Allows one to limit the number of encounters shown on the form entry tab of the patient dashboard specifically
 		</description>
 	</globalProperty>
+		<globalProperty>
+		<property>dashboard.observations.maximumNumberToShow</property>
+		<defaultValue>false</defaultValue>
```

Comment:
The above default value does not look valid.

Reply:
Changed